Program.cs

```diff
@@ -13,8 +14,22 @@ public static async Task Main()
                 // Get credentials
                 var config = new ServiceConfig(".env");
 
-                // Create the client
-                var client = RepositoryApiClient.CreateFromAccessKey(config.ServicePrincipalKey, config.AccessKey);
+            // Create the client
+            IRepositoryApiClient client;
+
+            if (config.AuthorizationType.Equals("LfdsUsernamePassword", StringComparison.OrdinalIgnoreCase))
+            {
+                client = RepositoryApiClient.CreateFromLfdsUsernamePassword(config.Username, config.Password, config.Organization, config.RepositoryId, config.BaseUrl);
+            }
+            else if (config.AuthorizationType.Equals("AccessKey", StringComparison.OrdinalIgnoreCase))
+            {
+                client = RepositoryApiClient.CreateFromAccessKey(config.ServicePrincipalKey, config.AccessKey);
+            }
+            else
+            {
+                Trace.TraceWarning("Invalid value for AUTHORIZATION_TYPE. It can only be 'AccessKey' or 'LfdsUsernamePassword'.");

```

Comment:
We switched to using `Console.WriteLine` instead of `Trace.TraceWarning`. Can you update this